packages/material-ui-lab/src/LoadingButton/LoadingButton.test.js

```diff
@@ -35,15 +26,13 @@ describe('<LoadingButton />', () => {
   });
 
   it('can be outlined', () => {
-    expect(() => {
-      render(
-        <LoadingButton
-          data-testid="root"
-          variant="outlined"
-          classes={{ outlined: 'loading-button-outlined' }}
-        />,
-      );
-    }).toErrorDev('The key `outlined` provided to the classes prop is not implemented');
```

Comment:
I wonder why we did expect an error here, when the tests is checking if the implementation works 😕

Reply:
Me too, it is kind of conflict. the test is `can be outlined` but does not accept `outlined` classes

Reply:
will fix this test and have the classes merge